music21/metadata/properties.py

```diff
@@ -1654,69 +1665,67 @@ class PropertyDescription:
         oldMusic21WorkId='orchestrator',
         valueType=Contributor,
         isContributor=True),
-
-    # The following mei property terms are not found in dcterms, marcrel, or humdrum.
-
-    PropertyDescription(
-        name='subtitle',
-        namespace='mei',
-        isContributor=False)
 )
 
 # -----------------------------------------------------------------------------
 # Dictionaries generated from STANDARD_PROPERTY_DESCRIPTIONS for looking up
 # various things quickly.
 
-NSKEY_TO_PROPERTYDESCRIPTION: dict = {
+NSKEY_TO_PROPERTY_DESCRIPTION: dict = {
     f'{x.namespace}:{x.name}':
         x for x in STANDARD_PROPERTY_DESCRIPTIONS}
 
-NSKEY_TO_VALUETYPE: dict = {
+NSKEY_TO_VALUE_TYPE: dict = {
     f'{x.namespace}:{x.name}':
         x.valueType for x in STANDARD_PROPERTY_DESCRIPTIONS}
 
-NSKEY_TO_CONTRIBUTORUNIQUENAME: dict = {
+NSKEY_TO_CONTRIBUTOR_UNIQUE_NAME: dict = {
     f'{x.namespace}:{x.name}':
         x.uniqueName if x.uniqueName
         else x.oldMusic21WorkId if x.oldMusic21WorkId
         else x.name
         for x in STANDARD_PROPERTY_DESCRIPTIONS if x.isContributor}
 
-NSKEY_TO_UNIQUENAME: dict = {
+NSKEY_TO_UNIQUE_NAME: dict = {
     f'{x.namespace}:{x.name}':
         x.uniqueName if x.uniqueName
         else x.oldMusic21WorkId if x.oldMusic21WorkId
         else x.name
         for x in STANDARD_PROPERTY_DESCRIPTIONS}
 
-UNIQUENAME_TO_NSKEY: dict = {
+UNIQUE_NAME_TO_NSKEY: dict = {
     x.uniqueName if x.uniqueName
     else x.oldMusic21WorkId if x.oldMusic21WorkId
     else x.name:
         f'{x.namespace}:{x.name}'
         for x in STANDARD_PROPERTY_DESCRIPTIONS}
 
-UNIQUENAME_TO_PROPERTYDESCRIPTION: dict = {
+UNIQUE_NAME_TO_PROPERTY_DESCRIPTION: dict = {
```

Comment:
Thanks for all these renames.  Our screen-reader editors will especially thank you.